lib/cisco_node_utils/interface.rb

```diff
@@ -114,6 +131,31 @@ def default_encapsulation_dot1q
       config_get_default('interface', 'encapsulation_dot1q')
     end
 
+    def encapsulation_profile_vni
+      val = config_get('interface', 'encapsulation_vni', @name)
+      debug "val from get is #{val}"
+      return '' if val.nil?
+      val.first.strip
+    end
```

Comment:
1. I don't see any yaml entries for `encapsulation_vni` in this diff or in the branch?
2. debug in the wild (use CiscoLogger.debug)
3. Should be able to just call config_get() by itself. The return and strip can be removed.